src/components/ReduxAppWrapper/store/reducers/captures.ts

```diff
@@ -32,7 +42,27 @@ export default function captures(
 
   if (action.type === constants.CAPTURE_CREATE) {
     const key = buildCaptureStateKey(action.payload)
-    return { ...state, [key]: action.payload }
+
+    // Record a new capture created
+    const { [key]: currentTakesHistory } = state.takesHistory
+    const newTakesHistory = [...currentTakesHistory, new Date().toISOString()]
+
+    const newPayload = {
+      ...action.payload,
+      sdkMetadata: {
+        ...action.payload.sdkMetadata,
+        // Write latest `take_number` to `sdkMetadata`
+        take_number: newTakesHistory.length,
+      },
+    }
+
+    return {
+      ...state,
+      [key]: newPayload,
+
+      // Update `takesHistory` with latest data
+      takesHistory: { ...state.takesHistory, [key]: newTakesHistory },
```

Comment:
Might be worth renaming the word `take` and `takes` to `frame` and `frames` so we know we are referring to the frames in multiframe capture features

Reply:
Answered in the above reply :point_up: